server/src/main/java/io/druid/guice/security/DruidAuthModule.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.guice.security;
+
+import com.fasterxml.jackson.databind.Module;
+import com.google.inject.Binder;
+import io.druid.guice.JsonConfigProvider;
+import io.druid.initialization.DruidModule;
+import io.druid.server.security.AuthConfig;
+
+import java.util.Collections;
+import java.util.List;
+
+public class DruidAuthModule implements DruidModule
+{
+  @Override
+  public List<? extends Module> getJacksonModules()
+  {
+    return Collections.emptyList();
+  }
+
+  @Override
+  public void configure(Binder binder)
+  {
+    JsonConfigProvider.bind(binder, "druid.auth", AuthConfig.class);
```

Comment:
Not sure if `request` should be included in the property name as the scope is much more than just requests to Druid. It can be called `druid.security` or something like that or can be kept as it is..